src/frequenz/client/base/channel.py

```diff
@@ -120,6 +138,40 @@ def parse_grpc_uri(
         parsed_uri.netloc if parsed_uri.port else f"{parsed_uri.netloc}:{defaults.port}"
     )
 
+    keep_alive = (
+        defaults.keep_alive.enabled
+        if options.keep_alive is None
+        else options.keep_alive
+    )
+    channel_options = (
+        [
+            ("grpc.http2.max_pings_without_data", 0),
+            ("grpc.keepalive_permit_without_calls", 1),
```

Comment:
I'm still curious why you are not making these configurable, I don't know if it is because you think is a bad idea or because it is not needed now?

Reply:
These are reasonable defaults, and it is unlikely that they will need to change.  I think it would be just too much information to make them configurable now, but we can revisit later if needed.